package.json

```diff
@@ -40,7 +40,7 @@
     "clean": "git clean -fX ./dist ./*.xpi ./addon.env && npm run :clean:doc ",
     "distclean": "git clean -dfX",
     "prepackage": "npm run-script clean && cross-env NODE_ENV=production npm run-script build",
-    "package": "echo TESTPILOT_ADDON_ID=${npm_package_id} > addon.env && echo TESTPILOT_ADDON_VERSION=${npm_package_version} >> addon.env && cd dist && nodezip -c ../addon.xpi *"
+    "package": "echo TESTPILOT_ADDON_ID=${npm_package_id} > addon.env && echo TESTPILOT_ADDON_VERSION=${npm_package_version} >> addon.env && cd dist && jpm xpi --dest-dir ../ && mv ../null.xpi ../addon.xpi"
```

Comment:
Instead of `cd dist`, I _think_ we can specify the `--addon-dir`, like this:

```json
jpm xpi --addon-dir=dist/ --dest-dir=./
```

Although the only way I can seemingly get around the odd `null.xpi` filename is if I copy the package.json file into dist/package.json.

Reply:
>I think we can specify the --addon-dir

Good catch. 👍